queue_services/entity-emailer/src/entity_emailer/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.11.0'  # pylint: disable=invalid-name
+__version__ = '3.0.0'  # pylint: disable=invalid-name
```

Comment:
jyoti has tagged the lear release with 3.0.0 and we're keeping all the backend the same version correct? I figured we'd keep the versions the same as what we're labeling them in zenhub because so far that's what we've done. It's not a large change obvs but it will be the first time we are live with bcomp incorp. I think if we don't do that then we should change the release ticket back to 2.12 to avoid confusion